2019-02-17 18:10:28,820 [http-nio-9199-exec-4] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Load balancer does not have available server for client: rabbit-auth-server
2019-02-17 18:10:28,862 [http-nio-9199-exec-4] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Load balancer does not have available server for client: rabbit-auth-server
2019-02-17 18:10:32,143 [http-nio-9199-exec-5] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Load balancer does not have available server for client: rabbit-auth-server
2019-02-17 18:10:32,146 [http-nio-9199-exec-5] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Load balancer does not have available server for client: rabbit-auth-server
2019-02-17 18:10:32,594 [http-nio-9199-exec-7] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Load balancer does not have available server for client: rabbit-auth-server
2019-02-17 18:10:32,799 [http-nio-9199-exec-8] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Load balancer does not have available server for client: rabbit-auth-server
2019-02-17 18:10:32,960 [http-nio-9199-exec-9] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Load balancer does not have available server for client: rabbit-auth-server
2019-02-17 18:10:33,567 [http-nio-9199-exec-7] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Load balancer does not have available server for client: rabbit-auth-server
2019-02-17 18:10:33,568 [http-nio-9199-exec-8] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Load balancer does not have available server for client: rabbit-auth-server
2019-02-17 18:10:33,568 [http-nio-9199-exec-9] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Load balancer does not have available server for client: rabbit-auth-server
2019-02-17 18:10:34,187 [http-nio-9199-exec-11] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Load balancer does not have available server for client: rabbit-auth-server
2019-02-17 18:10:34,188 [http-nio-9199-exec-11] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Load balancer does not have available server for client: rabbit-auth-server
2019-02-17 18:10:34,728 [http-nio-9199-exec-13] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Load balancer does not have available server for client: rabbit-auth-server
2019-02-17 18:10:34,729 [http-nio-9199-exec-13] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Load balancer does not have available server for client: rabbit-auth-server
2019-02-17 18:10:35,136 [http-nio-9199-exec-15] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Load balancer does not have available server for client: rabbit-auth-server
2019-02-17 18:10:35,137 [http-nio-9199-exec-15] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Load balancer does not have available server for client: rabbit-auth-server
2019-02-17 18:10:35,585 [http-nio-9199-exec-17] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Load balancer does not have available server for client: rabbit-auth-server
2019-02-17 18:10:35,586 [http-nio-9199-exec-17] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Load balancer does not have available server for client: rabbit-auth-server
2019-02-17 18:10:35,999 [http-nio-9199-exec-19] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Load balancer does not have available server for client: rabbit-auth-server
2019-02-17 18:10:35,999 [http-nio-9199-exec-19] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Load balancer does not have available server for client: rabbit-auth-server
2019-02-17 18:10:36,399 [http-nio-9199-exec-21] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Load balancer does not have available server for client: rabbit-auth-server
2019-02-17 18:10:36,399 [http-nio-9199-exec-21] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Load balancer does not have available server for client: rabbit-auth-server
2019-02-17 18:10:36,749 [http-nio-9199-exec-23] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Load balancer does not have available server for client: rabbit-auth-server
2019-02-17 18:10:36,750 [http-nio-9199-exec-23] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Load balancer does not have available server for client: rabbit-auth-server
2019-02-17 18:10:37,092 [http-nio-9199-exec-25] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Load balancer does not have available server for client: rabbit-auth-server
2019-02-17 18:10:37,093 [http-nio-9199-exec-25] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Load balancer does not have available server for client: rabbit-auth-server
2019-02-17 18:10:37,440 [http-nio-9199-exec-28] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Load balancer does not have available server for client: rabbit-auth-server
2019-02-17 18:10:37,441 [http-nio-9199-exec-28] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Load balancer does not have available server for client: rabbit-auth-server
2019-02-17 18:10:37,634 [http-nio-9199-exec-30] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Load balancer does not have available server for client: rabbit-auth-server
2019-02-17 18:10:37,635 [http-nio-9199-exec-30] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Load balancer does not have available server for client: rabbit-auth-server
2019-02-17 18:10:37,811 [http-nio-9199-exec-32] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Load balancer does not have available server for client: rabbit-auth-server
2019-02-17 18:10:37,812 [http-nio-9199-exec-32] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Load balancer does not have available server for client: rabbit-auth-server
2019-02-17 18:10:49,340 [http-nio-9199-exec-35] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Load balancer does not have available server for client: rabbit-auth-server
2019-02-17 18:10:49,341 [http-nio-9199-exec-35] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Load balancer does not have available server for client: rabbit-auth-server
2019-02-17 18:10:49,748 [http-nio-9199-exec-37] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Load balancer does not have available server for client: rabbit-auth-server
2019-02-17 18:10:49,749 [http-nio-9199-exec-37] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Load balancer does not have available server for client: rabbit-auth-server
2019-02-17 18:10:49,949 [http-nio-9199-exec-39] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Load balancer does not have available server for client: rabbit-auth-server
2019-02-17 18:10:49,951 [http-nio-9199-exec-39] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Load balancer does not have available server for client: rabbit-auth-server
2019-02-17 18:10:50,121 [http-nio-9199-exec-41] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Load balancer does not have available server for client: rabbit-auth-server
2019-02-17 18:10:50,122 [http-nio-9199-exec-41] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Load balancer does not have available server for client: rabbit-auth-server
2019-02-17 18:10:50,301 [http-nio-9199-exec-43] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Load balancer does not have available server for client: rabbit-auth-server
2019-02-17 18:10:50,302 [http-nio-9199-exec-43] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Load balancer does not have available server for client: rabbit-auth-server
2019-02-17 18:10:50,638 [http-nio-9199-exec-45] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Load balancer does not have available server for client: rabbit-auth-server
2019-02-17 18:10:50,638 [http-nio-9199-exec-45] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Load balancer does not have available server for client: rabbit-auth-server
2019-02-17 20:00:24,891 [http-nio-9199-exec-1] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Load balancer does not have available server for client: rabbit-auth-server
2019-02-17 20:00:24,917 [http-nio-9199-exec-1] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Load balancer does not have available server for client: rabbit-auth-server
2019-02-17 20:00:26,955 [http-nio-9199-exec-2] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Load balancer does not have available server for client: rabbit-auth-server
2019-02-17 20:00:26,957 [http-nio-9199-exec-2] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Load balancer does not have available server for client: rabbit-auth-server
2019-02-17 20:00:27,106 [http-nio-9199-exec-4] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Load balancer does not have available server for client: rabbit-auth-server
2019-02-17 20:00:27,107 [http-nio-9199-exec-4] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Load balancer does not have available server for client: rabbit-auth-server
2019-02-17 20:00:27,277 [http-nio-9199-exec-6] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Load balancer does not have available server for client: rabbit-auth-server
2019-02-17 20:00:27,279 [http-nio-9199-exec-6] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Load balancer does not have available server for client: rabbit-auth-server
2019-02-17 20:00:27,415 [http-nio-9199-exec-8] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Load balancer does not have available server for client: rabbit-auth-server
2019-02-17 20:00:27,415 [http-nio-9199-exec-8] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Load balancer does not have available server for client: rabbit-auth-server
2019-02-17 20:00:27,568 [http-nio-9199-exec-10] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Load balancer does not have available server for client: rabbit-auth-server
2019-02-17 20:00:27,569 [http-nio-9199-exec-10] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Load balancer does not have available server for client: rabbit-auth-server
2019-02-17 20:00:27,712 [http-nio-9199-exec-12] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Load balancer does not have available server for client: rabbit-auth-server
2019-02-17 20:00:27,713 [http-nio-9199-exec-12] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Load balancer does not have available server for client: rabbit-auth-server
2019-02-17 20:00:27,876 [http-nio-9199-exec-14] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Load balancer does not have available server for client: rabbit-auth-server
2019-02-17 20:00:27,877 [http-nio-9199-exec-14] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Load balancer does not have available server for client: rabbit-auth-server
2019-02-17 20:00:28,018 [http-nio-9199-exec-15] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Load balancer does not have available server for client: rabbit-auth-server
2019-02-17 20:00:28,019 [http-nio-9199-exec-15] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Load balancer does not have available server for client: rabbit-auth-server
2019-02-17 20:00:28,157 [http-nio-9199-exec-18] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Load balancer does not have available server for client: rabbit-auth-server
2019-02-17 20:00:28,158 [http-nio-9199-exec-18] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Load balancer does not have available server for client: rabbit-auth-server
2019-02-17 20:00:28,317 [http-nio-9199-exec-20] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Load balancer does not have available server for client: rabbit-auth-server
2019-02-17 20:00:28,318 [http-nio-9199-exec-20] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Load balancer does not have available server for client: rabbit-auth-server
2019-02-17 20:00:28,466 [http-nio-9199-exec-22] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Load balancer does not have available server for client: rabbit-auth-server
2019-02-17 20:00:28,467 [http-nio-9199-exec-22] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Load balancer does not have available server for client: rabbit-auth-server
2019-02-17 20:00:28,620 [http-nio-9199-exec-25] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Load balancer does not have available server for client: rabbit-auth-server
2019-02-17 20:00:28,620 [http-nio-9199-exec-25] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Load balancer does not have available server for client: rabbit-auth-server
2019-02-17 20:00:28,770 [http-nio-9199-exec-27] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Load balancer does not have available server for client: rabbit-auth-server
2019-02-17 20:00:28,771 [http-nio-9199-exec-27] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Load balancer does not have available server for client: rabbit-auth-server
2019-02-17 20:00:28,923 [http-nio-9199-exec-29] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Load balancer does not have available server for client: rabbit-auth-server
2019-02-17 20:00:28,924 [http-nio-9199-exec-29] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Load balancer does not have available server for client: rabbit-auth-server
2019-02-17 20:00:29,074 [http-nio-9199-exec-31] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Load balancer does not have available server for client: rabbit-auth-server
2019-02-17 20:00:29,075 [http-nio-9199-exec-31] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Load balancer does not have available server for client: rabbit-auth-server
2019-02-17 20:00:29,257 [http-nio-9199-exec-33] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Load balancer does not have available server for client: rabbit-auth-server
2019-02-17 20:00:29,257 [http-nio-9199-exec-33] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Load balancer does not have available server for client: rabbit-auth-server
2019-02-17 20:00:29,425 [http-nio-9199-exec-35] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Load balancer does not have available server for client: rabbit-auth-server
2019-02-17 20:00:29,426 [http-nio-9199-exec-35] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Load balancer does not have available server for client: rabbit-auth-server
2019-02-17 20:00:29,589 [http-nio-9199-exec-38] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Load balancer does not have available server for client: rabbit-auth-server
2019-02-17 20:00:29,589 [http-nio-9199-exec-38] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Load balancer does not have available server for client: rabbit-auth-server
2019-02-17 20:00:29,762 [http-nio-9199-exec-39] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Load balancer does not have available server for client: rabbit-auth-server
2019-02-17 20:00:29,763 [http-nio-9199-exec-39] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Load balancer does not have available server for client: rabbit-auth-server
2019-02-17 20:00:29,920 [http-nio-9199-exec-42] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Load balancer does not have available server for client: rabbit-auth-server
2019-02-17 20:00:29,921 [http-nio-9199-exec-42] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Load balancer does not have available server for client: rabbit-auth-server
2019-02-17 20:00:30,095 [http-nio-9199-exec-43] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Hystrix circuit short-circuited and is OPEN
2019-02-17 20:00:30,095 [http-nio-9199-exec-43] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Hystrix circuit short-circuited and is OPEN
2019-02-17 20:00:30,236 [http-nio-9199-exec-45] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Hystrix circuit short-circuited and is OPEN
2019-02-17 20:00:30,237 [http-nio-9199-exec-45] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Hystrix circuit short-circuited and is OPEN
2019-02-17 20:00:30,425 [http-nio-9199-exec-47] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Hystrix circuit short-circuited and is OPEN
2019-02-17 20:00:30,428 [http-nio-9199-exec-47] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Hystrix circuit short-circuited and is OPEN
2019-02-17 20:00:30,573 [http-nio-9199-exec-49] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Hystrix circuit short-circuited and is OPEN
2019-02-17 20:00:30,574 [http-nio-9199-exec-49] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Hystrix circuit short-circuited and is OPEN
2019-02-17 20:00:34,624 [http-nio-9199-exec-1] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Hystrix circuit short-circuited and is OPEN
2019-02-17 20:00:34,624 [http-nio-9199-exec-1] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Hystrix circuit short-circuited and is OPEN
2019-02-17 20:00:35,199 [http-nio-9199-exec-3] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Load balancer does not have available server for client: rabbit-auth-server
2019-02-17 20:00:35,199 [http-nio-9199-exec-3] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Load balancer does not have available server for client: rabbit-auth-server
2019-02-17 20:00:35,618 [http-nio-9199-exec-5] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Hystrix circuit short-circuited and is OPEN
2019-02-17 20:00:35,619 [http-nio-9199-exec-5] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Hystrix circuit short-circuited and is OPEN
2019-02-17 20:00:36,047 [http-nio-9199-exec-7] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Hystrix circuit short-circuited and is OPEN
2019-02-17 20:00:36,048 [http-nio-9199-exec-7] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Hystrix circuit short-circuited and is OPEN
2019-02-17 20:00:36,354 [http-nio-9199-exec-9] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Hystrix circuit short-circuited and is OPEN
2019-02-17 20:00:36,355 [http-nio-9199-exec-9] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Hystrix circuit short-circuited and is OPEN
2019-02-17 20:00:36,545 [http-nio-9199-exec-11] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Hystrix circuit short-circuited and is OPEN
2019-02-17 20:00:36,545 [http-nio-9199-exec-11] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Hystrix circuit short-circuited and is OPEN
2019-02-17 20:00:36,731 [http-nio-9199-exec-13] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Hystrix circuit short-circuited and is OPEN
2019-02-17 20:00:36,731 [http-nio-9199-exec-13] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Hystrix circuit short-circuited and is OPEN
2019-02-17 20:00:36,905 [http-nio-9199-exec-16] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Hystrix circuit short-circuited and is OPEN
2019-02-17 20:00:36,905 [http-nio-9199-exec-16] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Hystrix circuit short-circuited and is OPEN
2019-02-17 20:00:37,074 [http-nio-9199-exec-17] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Hystrix circuit short-circuited and is OPEN
2019-02-17 20:00:37,075 [http-nio-9199-exec-17] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Hystrix circuit short-circuited and is OPEN
2019-02-17 20:00:37,255 [http-nio-9199-exec-19] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Hystrix circuit short-circuited and is OPEN
2019-02-17 20:00:37,256 [http-nio-9199-exec-19] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Hystrix circuit short-circuited and is OPEN
2019-02-17 20:00:37,447 [http-nio-9199-exec-21] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Hystrix circuit short-circuited and is OPEN
2019-02-17 20:00:37,448 [http-nio-9199-exec-21] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Hystrix circuit short-circuited and is OPEN
2019-02-17 20:00:37,607 [http-nio-9199-exec-24] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Hystrix circuit short-circuited and is OPEN
2019-02-17 20:00:37,608 [http-nio-9199-exec-24] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Hystrix circuit short-circuited and is OPEN
2019-02-17 20:00:37,802 [http-nio-9199-exec-25] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Hystrix circuit short-circuited and is OPEN
2019-02-17 20:00:37,802 [http-nio-9199-exec-25] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Hystrix circuit short-circuited and is OPEN
2019-02-17 20:00:37,971 [http-nio-9199-exec-27] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Hystrix circuit short-circuited and is OPEN
2019-02-17 20:00:37,971 [http-nio-9199-exec-27] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Hystrix circuit short-circuited and is OPEN
2019-02-17 20:00:38,140 [http-nio-9199-exec-29] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Hystrix circuit short-circuited and is OPEN
2019-02-17 20:00:38,142 [http-nio-9199-exec-29] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Hystrix circuit short-circuited and is OPEN
2019-02-17 20:00:38,313 [http-nio-9199-exec-31] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Hystrix circuit short-circuited and is OPEN
2019-02-17 20:00:38,313 [http-nio-9199-exec-31] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Hystrix circuit short-circuited and is OPEN
2019-02-17 20:00:38,491 [http-nio-9199-exec-33] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Hystrix circuit short-circuited and is OPEN
2019-02-17 20:00:38,492 [http-nio-9199-exec-33] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Hystrix circuit short-circuited and is OPEN
2019-02-17 20:00:38,669 [http-nio-9199-exec-35] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Hystrix circuit short-circuited and is OPEN
2019-02-17 20:00:38,670 [http-nio-9199-exec-35] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Hystrix circuit short-circuited and is OPEN
2019-02-17 20:00:38,836 [http-nio-9199-exec-38] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Hystrix circuit short-circuited and is OPEN
2019-02-17 20:00:38,837 [http-nio-9199-exec-38] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Hystrix circuit short-circuited and is OPEN
2019-02-17 20:00:39,021 [http-nio-9199-exec-39] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Hystrix circuit short-circuited and is OPEN
2019-02-17 20:00:39,021 [http-nio-9199-exec-39] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Hystrix circuit short-circuited and is OPEN
2019-02-17 20:00:39,187 [http-nio-9199-exec-42] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Hystrix circuit short-circuited and is OPEN
2019-02-17 20:00:39,188 [http-nio-9199-exec-42] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Hystrix circuit short-circuited and is OPEN
2019-02-17 20:00:39,368 [http-nio-9199-exec-43] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Hystrix circuit short-circuited and is OPEN
2019-02-17 20:00:39,368 [http-nio-9199-exec-43] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Hystrix circuit short-circuited and is OPEN
2019-02-17 20:00:39,549 [http-nio-9199-exec-45] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Hystrix circuit short-circuited and is OPEN
2019-02-17 20:00:39,549 [http-nio-9199-exec-45] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Hystrix circuit short-circuited and is OPEN
2019-02-17 20:00:39,701 [http-nio-9199-exec-47] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Hystrix circuit short-circuited and is OPEN
2019-02-17 20:00:39,701 [http-nio-9199-exec-47] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Hystrix circuit short-circuited and is OPEN
2019-02-17 20:00:39,867 [http-nio-9199-exec-49] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Hystrix circuit short-circuited and is OPEN
2019-02-17 20:00:39,868 [http-nio-9199-exec-49] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Hystrix circuit short-circuited and is OPEN
2019-02-17 20:00:40,050 [http-nio-9199-exec-1] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Hystrix circuit short-circuited and is OPEN
2019-02-17 20:00:40,051 [http-nio-9199-exec-1] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Hystrix circuit short-circuited and is OPEN
2019-02-17 20:00:40,954 [http-nio-9199-exec-4] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Hystrix circuit short-circuited and is OPEN
2019-02-17 20:00:40,954 [http-nio-9199-exec-4] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:54 - 调用:rabbit-auth-server 异常：Hystrix circuit short-circuited and is OPEN
2019-02-17 20:01:18,946 [http-nio-9199-exec-11] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:57 - 调用:rabbit-auth-server 异常：认证授权模块不可用
2019-02-17 20:01:18,947 [http-nio-9199-exec-11] ERROR [c.r.gateway.componet.fallback.AuthFallbackProvider] AuthFallbackProvider.java:57 - 调用:rabbit-auth-server 异常：认证授权模块不可用
